gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/logback.xml

```diff
@@ -54,6 +54,7 @@
     <logger name="org.reflections" level="WARN" />
     <logger name="org.springframework" level="WARN" />
     <logger name="org.eclipse.jetty" level="WARN" />
+    <logger name="org.mongodb.driver.protocol.command" level="DEBUG" />
```

Comment:
**todo**: can you please revert or comment this line ?

Reply:
Well spot 😅